src/components/select/select.js

```diff
@@ -337,17 +338,41 @@ function SelectDirective($mdSelect, $mdUtil, $mdConstant, $mdTheming, $mdAria, $
       mdSelectCtrl.setLabelText = function(text) {
         mdSelectCtrl.setIsPlaceholder(!text);
 
-        if (attr.mdSelectedText) {
-          text = $parse(attr.mdSelectedText)(scope);
-        } else {
+        // Whether the select label has been given via user content rather than the internal
+        // template of <md-option>
+        var isSelectLabelFromUser = false;
+
+        if (attr.mdSelectedText && attr.mdSelectedHtml) {
+          throw Error('md-select cannot have both `md-selected-text` and `md-selected-html`');
+        }
+
+        if (attr.mdSelectedText || attr.mdSelectedHtml) {
+          text = $parse(attr.mdSelectedText || attr.mdSelectedHtml)(scope);
+          isSelectLabelFromUser = true;
+        } else if (!text) {
           // Use placeholder attribute, otherwise fallback to the md-input-container label
           var tmpPlaceholder = attr.placeholder ||
               (containerCtrl && containerCtrl.label ? containerCtrl.label.text() : '');
-          text = text || tmpPlaceholder || '';
+
+          text = tmpPlaceholder || '';
+          isSelectLabelFromUser = true;
         }
 
         var target = valueEl.children().eq(0);
-        target.html(text);
+
+        if (attr.mdSelectedHtml) {
+          if (!$injector.has('$sanitize')) {
+              throw Error('The ngSanitize module must be loaded in order to use ' +
+                  'md-treat-selected-text-as-html.');
```

Comment:
If there's no $sanitize, the user will get an [$sce:unsafe](https://docs.angularjs.org/error/$sce/unsafe) error from getTrustedHtml, which points to the sanitizer already. It can make sense to not have it (codesize), and rely on $sce.trustAsHtml only instead, and you'll fail hard for them at this point. So from a purely security POV I wouldn't throw here or look for $sanitize, and just let $sce.getTrustedHtml do its thing.